Test/BSN.Commons.Orm.EntityFrameworkCore.Tests/Infrastructure/DatabaseFactory.cs

```diff
@@ -0,0 +1,44 @@
+﻿using BSN.Commons.Test.Data;
```

Comment:
As I know we were going to refactor these codes. These files is very duplicate with EntityFramework.Tests and EntityFrameworkCore.Tests

Reply:
UnitTestContext is Depend on EntityFramework .Net/.NetCore Version.
RepositoryBase is Depend on .Net/.NetCore version.

So i have tried to do most. but it seems to be bad to separate UnitTest Entities from UnitTestContext.

Reply:
No, I think we must use generic class to resolve this issue, and move all classes to commons test for these classes.